docs/superpowers/specs/2026-07-29-wire-foretees-tee-times-via-booking-agent-design.md

```
@@ -0,0 +1,133 @@
+
+| Choice | Value |
+|---|---|
+| Cutover | Replace FastAPI scrape for listing |
+| Credentials | Per-user profile only |
+| SPA | Unchanged |
+| Layer | `ForeteesService.get_tee_times` proxies to agent |
+| Failures | Map agent `502` / transport errors to HTTP errors (not `[]`) |
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)